apps/framework-cli/src/framework/aggregations.ts

```diff
@@ -3,6 +3,9 @@ import {
   ClickHouseClient,
 } from "npm:@clickhouse/client-web@1.0.1";
 import { watch } from "npm:chokidar@3.6.0";
+import * as fastq from "npm:fastq@1.17.1";
```

Comment:
https://www.npmjs.com/package/fastq

Seems lightweight, recently maintained, & has lots of downloads. LMK if you have another package or approach in mind.